### PR TITLE
[Merged by Bors] - chore(GroupTheory/Perm): drop `DecidableEq`, `Fintype` -> `Finite`

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/GroupTheory/Perm/Cycle/Basic.lean [hunk 1]
```diff
@@ -1830,7 +1827,7 @@ open Equiv
 
 namespace List
 
-variable [DecidableEq α] {l : List α}
+variable [i4 : DecidableEq α] {l : List α}
```
Comment (tb65536) [inline Mathlib/GroupTheory/Perm/Cycle/Basic.lean:1830]: What's the motivation for this?

**Review by tb65536 — APPROVED** [review]

File: Mathlib/GroupTheory/Perm/Cycle/Basic.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/GroupTheory/Perm/Cycle/Basic.lean]: It was a temporary change to see which `[DecidableEq α]` variable is still available. Then I forgot to change back.
```suggestion
variable [DecidableEq α] {l : List α}
```

**Comment by tb65536** [comment]
maintainer merge

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**